src/coreclr/tools/Common/Compiler/DependencyAnalysis/Target_Riscv64/Register.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ILCompiler.DependencyAnalysis.Riscv64
+{
+    public enum Register
+    {
+        X0 = 0,
+        X1 = 1,
+        X2 = 2,
+        X3 = 3,
+        X4 = 4,
+        X5 = 5,
+        X6 = 6,
+        X7 = 7,
+        X8 = 8,
+        X9 = 9,
+        X10 = 10,
+        X11 = 11,
+        X12 = 12,
+        X13 = 13,
+        X14 = 14,
+        X15 = 15,
+        X16 = 16,
+        X17 = 17,
+        X18 = 18,
+        X19 = 19,
+        X20 = 20,
+        X21 = 21,
+        X22 = 22,
+        X23 = 23,
+        X24 = 24,
+        X25 = 25,
+        X26 = 26,
+        X27 = 27,
+        X28 = 28,
+        X29 = 29,
+        X30 = 30,
+
+        X31 = 31,
```

Comment:
Any reason for not to use [RISC-V ABI registers](https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-cc.adoc#integer-register-convention), that are more readable?